pages/docs/environments/gitpod.mdx

```diff
@@ -2,13 +2,66 @@ import DocsLayout from "@components/layouts/DocsLayout.js";
 
 ## Gitpod
 
-In the cloud free development environment which will have all the dependencies you need (for example MongoDB).
+Gitpod is an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.<br />
+It also automates the provisioning of a ready-to-code developer environment, so you will have all the dependencies you need (for example MongoDB).<br />
+With Gitpod, there’s no need to download any code to your machine. It can create a server-side-dev-environment for you in seconds from any repository on GitHub.<br />
```

Comment:
`<br />` this is not needed, it is best to use markdown for styling 

```suggestion
Gitpod is an open-source application that provides a prebuilt development environment in your browser - powered by VS Code. It also automates the provisioning of a ready-to-code developer environment, so you will have all the dependencies you need (for example MongoDB). With Gitpod, there’s no need to download any code to your machine. It can create a server-side-dev-environment for you in seconds from any repository on GitHub.
```